docs/products/mysql/howto/create-missing-primary-keys.rst

```diff
@@ -0,0 +1,79 @@
+Create missing primary keys
+===========================
+
+Find here strategies to create missing primary keys in your Aiven for MySQL service. First, you can determine which tables are missing primary keys by running:
```

Comment:
Maybe we can link back here saying "To understand the importance of primary keys for replication... check this article"

Reply:
Also I would state that the below is a query and needs to be run once connected to the mysql database

Reply:
I would also add two chapters: 
* List tables without primary key
* Add primary key to a table